<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>图像增强系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupbox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>原始图片/光照图/增强后图片</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>平滑度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="smoothnessSlider">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
           <property name="value">
            <number>301</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>亮度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="brightnessSlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>70</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>14</number>
         </property>
         <item>
          <widget class="QPushButton" name="loadBtn">
           <property name="text">
            <string>图片加载</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="enhanceBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>图片增强</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>保存图片</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="loadAct"/>
    <addaction name="saveAct"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>调整</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="aboutAct"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="saveAct">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="loadAct">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="aboutAct">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>待完成</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>待完成</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>loadAct</sender>
   <signal>triggered()</signal>
   <receiver>loadBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAct</sender>
   <signal>triggered()</signal>
   <receiver>saveBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>916</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadImg()</slot>
  <slot>enhanceImg()</slot>
  <slot>saveImg()</slot>
 </slots>
</ui>
